english/wake-up/wake-up-again/wake-up-again.md

```diff
@@ -1,11 +1,13 @@
 Thump. Thump. Thump.
 
-The sound woke you up from your slumber. Well, not really. You lay in 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.